### PR TITLE
Allowing for bypassing detune check during motor move

### PR DESCRIPTION
**Pull request by lisazacarias**
Also addressing issue #132

### CODE REVIEW

File: lcls_tools/superconducting/scLinac.py [hunk 1]
```diff
@@ -480,39 +480,48 @@ def move(
...
         :param changeLimits: whether to change the speed and steps
+        :param check_detune: whether to check for valid detune after each move
         :return:
         """
 
         self.check_abort()
+        maxSteps = abs(maxSteps)
 
         if changeLimits:
             # on the off chance that someone tries to write a negative maximum
```
Comment (nneveu) [inline lcls_tools/superconducting/scLinac.py:498]: Is a negative no longer a problem?

File: lcls_tools/superconducting/scLinac.py [hunk 2]
```diff
@@ -480,39 +480,48 @@ def move(
...
+                speed,
+                changeLimits=False,
+                check_detune=check_detune,
+            )
 
-    def issueMoveCommand(self, numSteps):
+    def issueMoveCommand(self, numSteps: int, check_detune: bool = True):
         # this is necessary because the tuners for the HLs move the other direction
         if self.cavity.cryomodule.is_harmonic_linearizer:
             numSteps *= -1
```
Comment (nneveu) [inline lcls_tools/superconducting/scLinac.py:527]: Maybe this is the answer to my q above. You check here now?

File: lcls_tools/superconducting/scLinac.py (hunk 1 above)
Comment (lisazacarias) [reply lcls_tools/superconducting/scLinac.py:498]: This just immediately turns max steps into a positive number regardless of what the user requested

File: lcls_tools/superconducting/scLinac.py (hunk 2 above)
Comment (lisazacarias) [reply lcls_tools/superconducting/scLinac.py:527]: No, this is numSteps, which is slightly different (I should probably figure out a way to rewrite this lol). It's the number of steps requested by the user vs the maximum number of steps allowed by the stepper

File: lcls_tools/superconducting/scLinac.py (hunk 1 above)
Comment (nneveu) [reply lcls_tools/superconducting/scLinac.py:498]: So this is the max steps _possible_ for all cavities, which is some default in utils.DEFAULT_STEPPER_MAX_STEPS? Is the default sometimes neg?

Comment (nneveu) [reply lcls_tools/superconducting/scLinac.py:498]: Lisa clarified, it's the maxSteps for one tuner movement.

**Review by nneveu — APPROVED** [review]
Thanks!

**Pull request merged**